### ISSUE TITLE
SA1313 is emitted for C# 10 positional record struct parameters

### ISSUE THREAD

**Issue by NoahStolk** [DotNetAnalyzers/StyleCopAnalyzers#3384]
When using C# 10 record structs in .NET 6 I am getting SA1313 'Parameter ... should begin with lower-case letter', even though parameters for positional records should be pascal-cased, just like record classes which were added in C# 9. I am using StyleCop 1.2.0-beta.354.

```cs
public record struct RecordStruct(string Param); // Warning for 'Param'

public record class RecordClass(string Param); // No warnings
```

Is development for supporting C# 10 being worked on? I understand if this is a bit early as C# 10 is still in preview.

Related: https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/3168
